src/contracts/extensions/price-adaptors/ScaledPriceAdaptor.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {AggregatorInterface} from '../../dependencies/chainlink/AggregatorInterface.sol';
+import {IScaledPriceAdaptor} from '../../interfaces/IScaledPriceAdaptor.sol';
+
+/**
+ * @title ScaledPriceAdaptor
+ * @author Aave Labs
+ * @dev Price Adaptor for Chainlink price feeds with non standard decimal USD feeds to 8 decimals.
+ */
+contract ScaledPriceAdaptor is IScaledPriceAdaptor {
```

Comment:
I think we should use `Adapter` instead of `Adaptor` -- we are using american english generally. it also aligns with [this](https://github.com/bgd-labs/aave-capo/tree/main/src/contracts)